.buildkite/scripts/test_synapse_port_db.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash -xe
+
+find . -name '*.pyc' -delete
+
+pip install psycopg2 coverage coverage-enable-subprocess
```

Comment:
We should probably use a virtualenv rather than installing into the system python.